source/common/upstream/eds.cc

```diff
@@ -173,11 +173,13 @@ void EdsClusterImpl::onAssignmentTimeout() {
   // TODO(vishalpowar) This is not going to work for incremental updates, and we
   // need to instead change the health status to indicate the assignments are
   // stale.
+  // TODO(snowp): This should probably just use xDS TTLs?
```

Comment:
I think we definitely need to fill in the logic here and elsewhere to turn the removal into "make this effectively an empty ClusterLoadAssignment".

Reply:
I think so too, though I figured we could do it incrementally for the various resource types. #13632 does it for RTDS

Reply:
Can you add some tracking issue for this? We also need to document the support status in user facing docs, otherwise we're going to have folks come along with "TTL isn't working for resource X".